cryptography/go/sdk/README.md

```diff
@@ -0,0 +1,37 @@
+# Dapr secrets management (Dapr SDK)
```

Comment:
Can we also provide a little narrative about what the scenario is? What is being encrypted/decrypted?

Reply:
Done